Data-Collection-Inventory/2021Q1_csvs/sub_program_is_part_of_program.csv

```diff
@@ -1,38 +1,21 @@
 sub_program_reference_id,program_reference_id
 ACTD,Allen Brain Map
-ADMBA,Allen Brain Map
-AHBA,Allen Brain Map
-AMBA,Allen Brain Map
-AMBCA,Allen Brain Map
-ASCA,Allen Brain Map
-BrainSpan,Allen Brain Map
-IvyGBM,Allen Brain Map
-NHP,Allen Brain Map
-R24 Ghosh,BICCN
```

Comment:
part of BICCN, keep - same for the other BICCN subprograms below

Reply:
Added BICCN-related grants and subprograms that don't have associated projects yet.